docs/source/gbd2017_models/risks/risk_exposure_template.rst

```diff
@@ -0,0 +1,53 @@
+.. _2017_risk_exposure_template:
+
+----------------------
+Risk Exposure Template

```

Comment:
This may need to be '====' instead of '----'. Do y'all agree?

Reply:
agreed!